src/components/Tables/Table.tsx

```diff
@@ -151,10 +153,19 @@ const Table: React.FC<Props> = ({
 
     const [filterText, setFilterText] = useState<FilterText>({});
 
-    const [selectCheckBoxes, setSelectCheckBoxes] = useState(
+    const [selectCheckBoxes, _setSelectCheckBoxes] = useState(
```

Comment:
can we keep this as `setSelectCheckBoxes` because I think we've set underscore at the start of variable names to mean unused variable.

Reply:
Also generally why would you rename it (like is there a reason why this is stylistically better)